test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
I would suggest either removing this file entirely or adding a test instead (which would be preferred).

Reply:
What would I be testing here, since the model is just the default model generated by rails?

Reply:
That's an excellent point. Maybe then this could be a place where you add a test where you assert things about your fixtures -- so you have a test that fails when/if your fixtures change and also serves as a reminder that you have those fixtures in place when writing other tests?

Reply:
Right now, I'm just creating new users as needed in the setup for each test, so there are no fixtures yet. I'll remove this test for now and can always add it back if the model gets customized or if we start using fixtures.

Reply:
100% thought you had added the fixtures in that file, not commented them out. Sorry about that!

Reply:
Given that this file has no tests, would there be an harm in removing it entirely?